docs/design/podgroup-statistics.md

```diff
@@ -0,0 +1,12 @@
+# PodGroup Statistics
+
+## Backgrounds
+
+Each time when podgroups states changed, the controller will update the statistics of podgroup of each state in the queue's status. And at the end of each scheduling session, the volcano scheduler will also update the allocated filed in queue's status to recored the amount of the amount of resources allocated. Both components use `UpdateStatus` api to update the queue status, which will cause conflict errors. When the controller encounter such an error, it will trigger `AddRateLimited` to push back the podgroup into work queue, resulting in accumulation of memory leak. See in issue #3597.
+
+## Alternative
+Currently the statistics of podgroup of eatch state are only used for display by vcctl, there is no need to be persisted in queue's status. So when users need to use `vcctl queue get -n [name]` or `vcctl list` to display queues and each state of podgroups in queue, vcctl should calculate podgroup statistics in client side and then display them. And we can export these statistics of podgroups in each state as metrics.  
```

Comment:
what is `eatch` mean?

Reply:
each state means podgroups in pending/running/unknown/inqueue/completed status

Reply:
<img width="234" alt="image" src="https://github.com/user-attachments/assets/040ae8e5-2eb0-45ca-9aa2-94f857a81435">

I mean there is a typo

Reply:
Oh sorry, didn't notice that there was a typo, now I have fixed it.